code/modules/mob/mob_defines.dm

```diff
@@ -42,6 +42,7 @@
 	var/ear_damage = 0		//Carbon
 	var/stuttering = 0		//Carbon
 	var/slurring = 0		//Carbon
+	var/cultslurring = 0	//Carbon
```

Comment:
Ah that's the reason why so many variables/procs and whatnot are triggered with a -1 and I've been confused to hell.